infrastructure/cluster/flux-v2/reana/reana-release.yaml

```diff
@@ -1,84 +1,90 @@
-#         cephfs_os_share_id: 1c8a0cef-988b-4614-b072-33f75f9a084c
-#         cephfs_os_share_access_id: b0a39b58-9b44-4961-b041-53978552970d
-#     components:
-#       reana_ui:
-#         enabled: true
-#       reana_db:
-#         enabled: true
-#     # db_env_config:
-#     #   REANA_DB_NAME: "vre"
-#     #   REANA_DB_PORT: "6600"
-#     #   REANA_DB_HOST: "dbod-vre.cern.ch"
-#       # There are two more environment variables that should be set in order
-#       # to connect to a database:
-#       # REANA_DB_USERNAME: containing the database user name.
-#       # REANA_DB_PASSWORD: containing the password for the user previously set.
-#       # Both environment variables should be set inside a Kubernetes secret:
-#       # `[release-name]-db-secrets`
-  
-#     ingress:
-#       enabled: false
-#         # tls:
-#         #   self_signed_cert: true
-#         #   secret_name: reana-tls-secret
-#         #   hosts:
-#         #     - vre-reana.cern.ch
-#         # annotations:
-#         #   cert-manager.io/cluster-issuer: letsencrypt
-#         #   kubernetes.io/ingress.class: traefik
-#         #   traefik.ingress.kubernetes.io/router.entrypoints: https
-#         #   traefik.ingress.kubernetes.io/router.tls: "true"
-#         # extra:
-#         #   - name: http
-#         #     annotations:
-#         #       kubernetes.io/ingress.class: traefik
-#         #       traefik.ingress.kubernetes.io/router.entrypoints: http
-#         #       traefik.ingress.kubernetes.io/router.middlewares: kube-system-redirect-scheme@kubernetescrd
-#     traefik:
-#       ports:
-#         web:
-#           nodePort: 30080
-#         websecure:
-#           nodePort: 30444 # changed from 30443
-#           tls:
-#             # without additional configs will use default insecure one
-#             enabled: true
-#       enabled: true
-#       rbac:
-#         enabled: true
-#       service:
-#         type: NodePort
+
+  values:
+
+    shared_storage: 
+      backend: cephfs
+      volume_size: 1000
+      access_modes: ReadWriteMany
+      cephfs:
+        provisioner: manila-provisioner
+        type: "Meyrin CephFS"
+        availability_zone: nova
+        os_secret_name: os-trustee
+        os_secret_namespace: kube-system
+        # get shares with `openstack share list`, `openstack share show share_id`
+        cephfs_os_share_id: 1c8a0cef-988b-4614-b072-33f75f9a084c
+        cephfs_os_share_access_id: b0a39b58-9b44-4961-b041-53978552970d
+
+    components:
+
+      reana_ui:
+        enabled: true
+        local_users: false 
+      reana_db:
+        enabled: false
+      reana_server:
+        environment:
+          REANA_USER_EMAIL_CONFIRMATION: false
+        
+    db_env_config:
+      # REANA_DB_NAME: "reana"
+      # REANA_DB_PORT: "6600"
+      # REANA_DB_HOST: "dbod-vre.cern.ch"
+      
+      REANA_DB_NAME: "admin"
+      REANA_DB_PORT: "6627"
+      REANA_DB_HOST: "dbod-reana-vre.cern.ch"
```

Comment:
is this another DBOD instance? We should consider just using one instance.

Reply:
changed the connection to be with the same instance as the rucio one, I had this for ease of use as the other one is giving some problmes, but working on them to solve them and have a clean initial config. Changed the release accordingly, to connect to `dbod-vre.cern.ch`.